docs/pnpm.md

```diff
@@ -216,6 +218,11 @@ npm_translate_lock(
 )
 ```
 
+Patching applied by rules_js may slightly deviate from pnpm patching behavior due to the use
+of the `patch` command instead of the internal patching mechanism used by pnpm.
```

Comment:
Can you provide more details about which `patch` command is used? I can't recall if we're relying on a non-hermetic local patch tool on the machine here or a hermetic patch tool that comes with bazel

Reply:
It actually delegates to a bazel-lib util to do it so maybe I can just link to docs there.

Reply:
The implementation of that util has a if / else that ends up using different tools if I recall correctly. rules_js may be always picking one of those code paths because of the constraints of extracting to a directory